setup/templates/steps/config.html

```diff
@@ -93,7 +93,7 @@
   It is not possible to use the API without an API token.</p>
 
   <div class="form-group">
-      <input type="checkbox" name="api_enabled" value="false" id="api" >
+      <input type="checkbox" name="api_enabled" value="true" id="api" >
```

Comment:
This should be "false", to have the API disabled by default, shouldn't it?

Reply:
No. That is handled by the JavaScript. I don't understand it. When testing it again, I see that the value `API=false` is always written in the docker-compose.yml. I will check why this happens.

You can actually test the PR with this docker-compose.yml
```
# This file is used to run the mailu/setup utility

version: '3.6'

services:
  redis:
    image: redis:alpine
    restart: always

  development:
    image: ${DOCKER_ORG}/setup:${DEVELOPMENT}
    ports:
      - "0.0.0.0:1234:80"
    env_file: .env
    environment:
      stable_version: ${DEVELOPMENT}
      this_version: ${DEVELOPMENT}
      VERSIONS: ${DEVELOPMENT}
    depends_on:
      - redis
    restart: always
```
and this .env file
```
# Current versions
#DEVELOPMENT=local
DEVELOPMENT=pr-2638
#DOCKER_ORG=mailu
DOCKER_ORG=mailuci
```
You can switch the comments in the .env file to switch to a locally build setup. By default local images are build with docker organisation `mailu` and tag `local`.

Reply:
Found it. I forgot to add the attribute `id="api_enabled" ` to the checkbox (or I removed it by mistake.)